### PR TITLE
Fixed issues for purestorage solution offer

### PR DESCRIPTION
**Pull request by skeerthivasan**
Required items, please complete
   
   Change(s):
Initial Solution release for Pure Storage

   Reason for Change(s):
Adding Parser only solution package for PureStorage

   Version Updated:
No - First release

   Testing Completed:
Yes

   Checked that the validations are passing and have addressed any issues that are present:
Yes

### CODE REVIEW

File: Solutions/Pure Storage/SolutionMetadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "publisherId": "purestorageinc1707205060945",
```
Comment (v-atulyadav) [inline Solutions/Pure Storage/SolutionMetadata.json:2]: Hi @skeerthivasan,
In your partner center we have seen `purestoragemarketplaceadmin` publisherid. Please replace this and repackage again. Thanks

**Comment by v-atulyadav** [comment]
Hi @skeerthivasan,
Please repackage this solution with the help of below link. Thanks
[Azure-Sentinel/Tools/Create-Azure-Sentinel-Solution/V3/README.md at master · Azure/Azure-Sentinel (github.com)](https://github.com/Azure/Azure-Sentinel/blob/master/Tools/Create-Azure-Sentinel-Solution/V3/README.md)

**Comment by skeerthivasan** [comment]
> Hi @skeerthivasan, Please repackage this solution with the help of below link. Thanks [Azure-Sentinel/Tools/Create-Azure-Sentinel-Solution/V3/README.md at master · Azure/Azure-Sentinel (github.com)](https://github.com/Azure/Azure-Sentinel/blob/master/Tools/Create-Azure-Sentinel-Solution/V3/README.md)

@v-atulyadav , I have repackaged and pushed it again, but I see few validations are failed, can we restart the check again ? or Am I missing something ?

**Comment by skeerthivasan** [comment]
@v-atulyadav , I have pushed it again, checks are fine now. Please check and proceed

**Comment by skeerthivasan** [comment]
@v-atulyadav , I see you have repackaged it, below 'steps' line should be [], instead of null. This would cause issue during package upload in solution offer. Let me know your thoughts on this.
https://github.com/Azure/Azure-Sentinel/pull/10120/commits/ee4672ef799fea8a8182d920b0af5c381d7b482a#diff-1386cca6740fbce10529e79de2fb77a84307add596cbd654fbeba11f2fa9d4aeR53

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**